packages/core/src/configuration/Configuration.ts

```diff
@@ -477,7 +470,39 @@ export class Configuration {
    * This setting only works in environments that supports persisted storage.
    * There is also a performance penalty of extra IO/serialization.
    */
-  public usePersistedQueueStorage = false;
+  public usePersistedQueueStorage: boolean = false;
+
+  /**
+   * Gets or sets a value indicating whether to automatically send session start,
+   * session heartbeats and session end events.
+   */
+  public sessionsEnabled = false;
+
+  /**
+   * Internal property used to track the current session identifier.
+   */
+  public currentSessionIdentifier: string | null = null;
+
+  /**
+   *
+   * @param sendHeartbeats Controls whether heartbeat events are sent on an interval.
+   * @param heartbeatInterval The interval at which heartbeats are sent after the last sent event. The default is 1 minutes.
+   * @param useSessionIdManagement Allows you to manually control the session id. This is only recommended for single user desktop environments.
+   */
+  public useSessions(sendHeartbeats: boolean = true, heartbeatInterval: number = 60000, useSessionIdManagement: boolean = false) {
```

Comment:
@ejsmith I'm not sure about useSessionIdManagement, I guess this is the safe default since this can be used on node.

Reply:
It seems fine.